README.md

```diff
@@ -1,48 +1,147 @@
-# Familia
-An app to track family's required groceries
-
-### User Stories
-- As a user I want to create an account to be able to login.
-- As a user I want to access my existing account by logging in.
-- As a user I need to create my new family to have a list.
-- As a user I want to be able to join an existing family. 
-- As a user I need to view my family-list.
-- As a user I want to be able to add a new item to my list.
-
-### File structure 
+# [Familia]()
+
+### What is Familia?
+
+Familia is a shopping list, sharing and tracking app that aims to connect your household's needs with all of your family members. With Familia you'll never have to worry about forgetting what your fridge lacks or your mom needs.
+
+### User Stories: what can you do on our app?
+
+- As a user you want to create an account to be able to login.
+- As a user you want to access your existing account by logging in.
+- As a user you need to create your new family to have a list.
+- As a user you want to be able to join an existing family.
+- As a user you need to view your family-list.
+- As a user you want to be able to add a new item to your list.
+
+### How to run our app on your local machine?
+
+**Note: this App uses MongoDB as its database.*
+
+**Follow these instruction in order:**
+
+1- Open your terminal.
+
+2- Clone this repository.
+
+Using HTTP: `git clone https://github.com/FACK1/Familia.git`
+Using SSH: `git clone git@github.com:FACK1/Familia.git`
+
+3- Go to [mLab](https://mlab.com/), and make an account. Verify your email, and login. \*_Note: mLab is a hosting service for MongoDB databases._
+
+4- Press on `create new +`, select the free hosting plan `Sanbox`, press `continue`, then select your preffered region.
+
+5- Name your database something unique, and press `submit order`.
+
+6- Press on your database, and under the tab `users`, press `add database user`.
+
+7- Select a username, a password, and **make sure to save them somewhere, we're gonna need them for step 11**. Note: don't tick the `read only` option.
+
+8- Open the repo in your text editor, and your terminal too.
+
+9- Install dependencies and dev dependencies; first by inititalizing a package.json by typing in the terminal `npm init` and then `npm i`.
```

Comment:
You don't need to npm init if you've cloned. There will already be a package.json.